### DISCUSSION TITLE
different recombination / mutation rates in subpopulations

### DISCUSSION THREAD

**Discussion by andrei-barysenka** [tskit-dev/tskit#2159] [Q&A] [answered]
is there a way in msprime to simulate ancestry / mutations with demographic events (like a split) where individual populations have different recombination rates and mutation rates ?

**Comment by jeromekelleher** [comment, marked as answer]
Unfortunately not @andrei-barysenka. We have an existing issue (#1095) discussing the idea of changing recombination rates over time.

A per-population mutation rate is an interesting idea and could be done in ``sim_mutaitons`` by keeping track of what population an edge is "in" (but I guess we'd need to keep track of all migrations as well in a full-arg like way).

Neither would be particularly straightforward to implement, I'm afraid.
